### ISSUE TITLE
[Bug]: Still getting workflow failure notifications

### ISSUE THREAD

**Issue by jsjoeio** [coder/code-server#6090]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS/Web Information

N/A

### Steps to Reproduce

N/A

### Expected

I shouldn't get them

### Actual

I get them lol

### Does this issue happen in VS Code or GitHub Codespaces?

- [X] I cannot reproduce this in VS Code.
- [X] I cannot reproduce this in GitHub Codespaces.

### Are you accessing code-server over HTTPS?

- [X] I am using HTTPS.

### Notes

@code-asher do you mind making a commit to https://github.com/coder/code-server/blob/main/.github/workflows/security.yaml ? I think because I was the last non-bot to edit this file, GitHub pings me when it fails.

Context: https://github.com/orgs/community/discussions/50508

**Comment by jsjoeio** [comment]
So this didn't fix it sadly but I reached out to GitHub and they said this:

> The expected behavior for scheduled workflows in public repositories is that workflow notifications are sent to the user who initially created the workflow or the user who last edited the cron syntax in the workflow file.
> 
> https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/notifications-for-workflow-runs
> 
> The engineering team is currently working on a different approach in order to fix the issue for users who no longer belong to the repository.
> 
> In the meantime could you reach out to the team to confirm who the GITHUB ACTOR is on the workflow file? They can confirm this by running echo $GITHUB_ACTOR in your workflow file, If it is still set to your username, then they'll need to change it by either disabling and re-enabling the workflow or they'll need someone else to edit the cron section of the workflow file.
> 
> You mentioned the current maintainer made a change to the file but you still get notified when one of the jobs failed. Could you confirm if they made a change to the cron section of the file? They'll have to make a change there or either disable and re-enable the workflow.
> 
> See the guide below on how to disable and re-enable workflows:
> 
> https://docs.github.com/en/actions/managing-workflow-runs/disabling-and-enabling-a-workflow

TLDR apparently you have to edit the cron syntax part of the workflow or disable and re-enable 😅

**Comment by code-asher** [comment]
I disabled/enabled the workflow so hopefully that works this time! 
If not I will try editing the cron section.
